src/main.cpp

```diff
@@ -1,6 +1,10 @@
-//
-// Created by ricardo on 05-05-2023.
-//
-int main(){
+#include "menu.h"
+
+int main() {
+    std::string const nodesFilePath = "../dataset/Real-world Graphs/graph1/nodes.csv";
```

Comment:
We can try further ahead to store each directory and then load each graph at run-time (Later issue)

Reply:
We can, but we need to do some testing because that may really slow down the program at startup, and we might not use all the graphs in the program at once